stellarphot/photometry/profiles.py

```diff
@@ -245,17 +247,32 @@ def pixel_values_in_profile(self):
         """
         radii = []
         pixel_values = []
-        for rad, ap in zip(
-            self.radial_profile.radius, self.radial_profile.apertures, strict=True
-        ):
+        for ap in self.radial_profile.apertures:
+            # Calculate the distance of each pixel from the center
+            grid_x, grid_y = np.mgrid[: self._data.shape[0], : self._data.shape[1]]
+            dist_from_cen = np.sqrt(
+                (grid_x - self._cen[1]) ** 2 + (grid_y - self._cen[0]) ** 2
```

Comment:
Is this correct, you have `grid_x` determined by `: self._data.shape[0]` but you compute the distance from center using `(grid_x - self._cen[1])`?

Reply:
Yeah; this is the array vs image thing.